cmake/BuildLibElf.cmake

```diff
@@ -0,0 +1,15 @@
+include(ExternalProject)
+
+ExternalProject_Add(libelf
+  PREFIX "vendor/elfutils"
+  URL http://archive.ubuntu.com/ubuntu/pool/main/e/elfutils/elfutils_0.163.orig.tar.bz2
```

Comment:
that won't work for us, since this type of 'during the make' fetching is impossible in dc environment. All source has to be present during the build. We cannot do uploads during the build. Sorry. I guess we have to live with libelf.so or ask people build one independently as libelf_pic.a with -fpic